graphs/heat_map.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+from datetime import datetime, timedelta
+import argparse
+import itertools
+import numpy as np
+import yaml
+from yaml import CLoader as Loader
+from plotly.graph_objs import Figure, Layout, Scatter
+import plotly.express as px
+
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+
+parser = argparse.ArgumentParser(description='COVID-19 visualization for Slovakia')
+parser.add_argument('simulated',
+                    metavar='simulated',
+                    type=str,
+                    help=f"YAML file with simulation results")
+args = parser.parse_args()
+
+with open(args.simulated, 'r') as stream:
+    try:
+        data = yaml.load(stream, Loader=Loader)
+    except yaml.YAMLError as exc:
+        raise exc
+
+    best_errors = {}
+    for group in data:
+        if ("results" not in group) and ("result_abbrev" not in group):
+            continue
+
+        prefix_length = group["params"]["prefix_length"]
+        b0 = group["params"]["b0"]
+        gamma2 = group["params"]["gamma2"]
+
+        if gamma2 not in best_errors:
+            best_errors[gamma2] = {}
+
+        if "result_abbrev" in group:
+            error_sum = group["result_abbrev"]["error"]
+        else:
+            error_sum = sum(result["error"] for result in group["results"])
+        best_errors[gamma2][(b0, prefix_length)] = error_sum
```

Comment:
```suggestion
        best_errors.setdefault(gamma2, {})[(b0, prefix_length)] = error_sum
```

A vyššie treba zmazať
```python
        if gamma2 not in best_errors:
￼            best_errors[gamma2] = {}
```

Reply:
Done.